LICENSE

```diff
@@ -1,7 +1,10 @@
-Copyright © 2018 WHATWG (Apple, Google, Mozilla, Microsoft).
-This work is licensed under a Creative Commons Attribution 4.0 International License:
+Copyright © 2020 WHATWG (Apple, Google, Mozilla, Microsoft).
+This work is licensed under a Creative Commons Attribution 4.0 International License.
```

Comment:
Existing problem, but should this say Public License as per https://creativecommons.org/licenses/by/4.0/legalcode.txt?

Reply:
I don't think so, this matches the recommended wording at https://creativecommons.org/choose/results-one?license_code=by&amp;jurisdiction=&amp;version=4.0&amp;lang=en.